src/elements/DataTable.vue

```diff
@@ -7,11 +7,25 @@
     </caption>
     <thead>
       <tr>
-        <th v-for="(col, index) in parsedColumns" scope="col">{{ displayName(col) }}</th>
+        <th v-for="(col, index) in parsedColumns" scope="col">
+          <lux-icon-base v-if="col.sortable" width="16" height="16" icon-name="approved">
```

Comment:
I'm confused by the icon name here.  Sould sort be better than approved?

Reply:
Nice catch, @carolyncole! That's an artifact from an early experiment. I'll fix.